source/developers/concepts/hooks_extension_framework.rst

```diff
@@ -163,6 +163,38 @@ Filters are useful when you need an immediate response that directly modifies th
 
 We encourage you to review the list of use cases for events and filters to draw inspiration from real-life scenarios and see if your use case aligns with any of them. Also, maybe your feature can be implementing using the framework but there's not an available event or filter for it yet. In that case, consider proposing a new event or filter to the community!
 
+What's Next?
+============
+
+Contribute to the Hooks Extension Framework
```

Comment:
I think I would not nest this in a "What's Next" - I would just have this be its own second-level header so it shows up in the sidebar (so with `****` below)